### ISSUE TITLE
Improve accuracy, speed and architecture ideas.

### ISSUE THREAD

**Issue by xbankov** [video699/implementation-system#6]
- [ ] Choose TensorFlow or PyTorch, not both, it is inconvenient.
- [ ] Use some pre-trained models for siamese page detector.
- [ ] Rewrite fastai1 to fastai2 or use basic PyTorch for semantic segmentation screen detector.
- [ ] Successor to VGG16, e.g. xResNet features could be used as a page detector.
- [ ] Train GAN for screen to slide restoration translation.
